src/Service/FileBasedAttestationCertificateTrustStore.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Copyright 2019 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace App\Service;
+
+use App\Entity\PublicKeyCredentialSource;
+use App\Exception\AttestationStatementNotFoundException;
+use Webauthn\AttestationStatement\AttestationStatement;
+
+class FileBasedAttestationCertificateTrustStore implements AttestationCertificateTrustStore
```

Comment:
Simple as this class is, it should be unit tested

Reply:
Yes, different branch with more logic